docs/source/user-guide/cli.md

```diff
@@ -191,7 +191,7 @@ DataFusion CLI v16.0.0
 2 rows in set. Query took 0.007 seconds.
 ```
 
-## Creating external tables
+## Creating External Tables
```

Comment:
drive by cleanup -- the other headings are capitalized so it seemed strange that this one was not